### DISCUSSION TITLE
Is Slint good for creating a code editor?

### DISCUSSION THREAD

**Discussion by megaredb** [slint-ui/slint#1823] [Q&A] [answered]
Is Slint good for creating a crossplatform code editor? I want build a very fast, with low RAM usage, code editor in Rust.
Or should I look at other frameworks?

**Comment by ogoffart** [comment, marked as answer]
This is one use case that Slint aim at doing. 
Slint indeed aim to be fast and lightweight.
Slint is probably still missing some features for good desktop experience. But not all hope is lost as these feature can always be implemented.
So it all depends what feature you need.
